standards/development-principles.md

```diff
@@ -0,0 +1,93 @@
+---
+category: Development Principles
+expires: 2019-03-31
```

Comment:
If we think these principles haven't been updated for a while and need some attention, should we make these dates sooner?

I'm not sure who's responsible for keeping track of which guidance is expiring soon and needs looking at, but that's a separate conversation...

Reply:
That date was more a finger in the air deadline but I agree these should be reviewed more often and reflect current trends in the industry. Again this is something we should discuss as a community.